eclair-node/src/main/scala/fr/acinq/eclair/api/ExtraDirectives.scala

```diff
@@ -20,13 +20,14 @@ import akka.http.scaladsl.marshalling.ToResponseMarshaller
 import akka.http.scaladsl.model.StatusCodes.NotFound
 import akka.http.scaladsl.model.{ContentTypes, HttpResponse}
 import akka.http.scaladsl.server.{Directive1, Directives, MalformedFormFieldRejection, Route}
-import fr.acinq.bitcoin.ByteVector32
+import fr.acinq.bitcoin.{ByteVector32, ByteVector64}
```

Comment:
nit: remove changes from this file? ctrl + alt + o

Reply:
Done